src/view/renderer.js

```diff
@@ -177,6 +185,10 @@ export default class Renderer {
 	 * removed as long selection is in the text node which needed it at first.
 	 */
 	render() {
+		if ( this.isComposing ) {
+			return;
```

Comment:
For sure it need some tests with collaboration begore merging this PR because it can causes more bad than good.